calcite-qml/demo/calite_test.pro

```diff
@@ -0,0 +1,77 @@
+# Copyright 2012-2021 Esri
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+mac {
+    cache()
+}
+
+#-------------------------------------------------------------------------------
+
+CONFIG += c++14
+
+# additional modules are pulled in via arcgisruntime.pri
+QT += opengl qml quick quickcontrols2
+
+ARCGIS_RUNTIME_VERSION = 100.13
+include($$PWD/arcgisruntime.pri)
+
+TEMPLATE = app
+TARGET = calite_test
+
+equals(QT_MAJOR_VERSION, 5) {
+    lessThan(QT_MINOR_VERSION, 15) { 
+        error("$$TARGET requires Qt 5.15.2")
+    }
+	equals(QT_MINOR_VERSION, 15) : lessThan(QT_PATCH_VERSION, 2) {
+		error("$$TARGET requires Qt 5.15.2")
+	}
+}
+
+equals(QT_MAJOR_VERSION, 6) {
+  error("This version of the ArcGIS Runtime SDK for Qt is incompatible with Qt 6")
+}
+
+#-------------------------------------------------------------------------------
+
+QML_IMPORT_PATH += /Users/neil9879/codebases/calcite-qml
```

Comment:
❓ can this be generalized? Or perhaps raise a qmake error if the path can't be found? Users will presumably need to update this.